__tests__/server.test.js

```diff
@@ -232,17 +232,17 @@ describe.skip("/api/workouts/:level", () => {
                 })
             })
         })
-        test("400: Responds with a bad request message if ID is invalid", () => {
+        test.skip("400: Responds with a bad request message if workout level is invalid", () => {
             return request(app)
-            .get("/api/workouts/out_id")
+            .get("/api/workouts/wrongLvl")
             .expect(400)
             .then((response) => {
-                expect(response.body.message).toBe("Invalid ID")
+                expect(response.body.message).toBe("Invalid query")
             })
         })
         test("404: Responds with a not found message if workout does not exist", () => {
             return request(app)
-            .get("/api/workouts/673b26e3656d6301098761e8")
+            .get("/api/workouts/44333")
```

Comment:
I don't think this is the correct format for a MongoDB ObjectId. If you're really struggling to come up with a MongoDB ObjectId yourself, you could either look into the database, find the final item of the collection, then add 1 to the number at the end, or just ask ChatGPT to generate a random MongoDB ObjectId string for you.

Reply:
Wait, my bad, we don't want it to be an ID because now we're doing it by level...